src/models/common/channel.rs

```diff
@@ -13,6 +13,7 @@ pub trait HasChannelInfo {
 #[derive(Debug, PartialEq, Clone, Serialize, Deserialize, Builder)]
 pub struct SlackChannelInfo {
     pub id: SlackChannelId,
+    pub name: String,
```

Comment:
Looking at examples, for some channels it doesn't exist, unfortunately, so this is need to be optional

Reply:
Oh really? All the events including the one you send seem to have it. But I will make it Optional too :)

Reply:
done